website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -137,33 +120,828 @@ Your private GitHub repo should have a clusters/management folder that includes
+[Helm Chart](../../references/helm-reference.md).
+Follow the instructions in the next section in order to configure RBAC correctly.
+
+:::tip
+To remove the emergency user as a login method, set the following values in the
+[Helm Chart](../../references/helm-reference.md):
+
+```yaml
+#
+adminUser:
+  create: false
+#
+additionalArgs:
+- --auth-methods=oidc
+#
+```
+
+If you are disabling an already existing emergency user, you will need to
+manually delete the Kubernetes Secret and any User Roles that were created on
+the cluster.
+:::
+
+</TabItem>
+</Tabs>
+
+### GitOps Dashboard Service Account Permissions
+
+This section covers the service account [permissions](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+for the Weave GitOps application, which the WGE UI requires to work. The default permissions are defined in the [Helm chart](https://github.com/weaveworks/weave-gitops/tree/main/charts/gitops-server/templates/role.yaml), which will generate a cluster role with the following permissions:
+
+```yaml
+rules:
+# Used to query the cluster
+- apiGroups: [""]
+  resources:  ["users", "groups"]  # set by rbac.impersonationResources
+  verbs: [ "impersonate" ]
+  # resourceNames: []              # set by rbac.impersonationResourceNames
+# Used to get OIDC/static user credentials for login
+- apiGroups: [""]
+  resources: [ "secrets" ]
+  verbs: [ "get", "list" ]
+  resourceNames:                  # set by rbac.viewSecretsResourceNames
+    - "cluster-user-auth"
+    - "oidc-auth"
+# The service account needs to read namespaces to know where it can query
+- apiGroups: [ "" ]
+  resources: [ "namespaces" ]
+  verbs: [ "get", "list" ]
+```
+
+These allow the pod to do three things:
+* [Impersonate](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#user-impersonation) the user and operate in the cluster as them
+* Read the available namespaces; this is required to understand users' permissions
+* Read the `cluster-user-auth` and `oidc-auth` secrets, the default secrets
+  to store the emergency cluster user account and OIDC configuration (see
+  [securing access to the dashboard](#securing-access-to-the-dashboard))
+
+### The Helm Values
+
+| Value                             | Description                                                         | Default                              |
+|-----------------------------------|---------------------------------------------------------------------|--------------------------------------|
+| `rbac.impersonationResources`     | Which resource types the service account can impersonate            | `["users", "groups"]`                |
+| `rbac.impersonationResourceNames` | Specific users, groups or services account that can be impersonated | `[]`                                 |
+| `rbac.viewSecretsResourceNames`   | Specific secrets that can be read                                   | `["cluster-user-auth", "oidc-auth"]` |
```

Comment:
These values aren't supported in the EE helm chart right now.

Reply:
@foot Huh, I got them from https://docs.gitops.weave.works/docs/configuration/service-account-permissions/#the-helm-values. So I should remove this "The Helm values" section?

Reply:
Yes!

Reply:
Done